NoSoft-SA/nspack
# Load staging overview

Summary of this change:

added feature for load staging on the loads grid


CHANGELOG entry:

### Added
- [added load staging overview](https://github.com/NoSoft-SA/nspack/commit/d7f76f85a3707feb5a5ed054ceba22b828f98f30)
- [added vw_hierarchical_locations](https://github.com/NoSoft-SA/nspack/commit/d728997c78b8542e07aa14a6118b6d246501d1c8)
### Changed
- [renamed descendants_for_ancestor_id](https://github.com/NoSoft-SA/nspack/commit/03846fa9fb7765bd16d75d0dd58414892d5c5c48)
- [refactored find_location_by](https://github.com/NoSoft-SA/nspack/commit/8c8a6e4b3d7b63d33f4cc30867b72d66ca0b3a37)
- [refactored for_select_locations](https://github.com/NoSoft-SA/nspack/commit/d75ae04dfbb41d818c26d0fb10223504b9c953c4)
- [removed find_location_by_location_long_code method](https://github.com/NoSoft-SA/nspack/pull/360/commits/77b722280277e800b95be9fc4a652322672255bc)
### Fixed
- [moved storage_rule_factory](https://github.com/NoSoft-SA/nspack/pull/360/commits/f72353102cd025cbc0b5b18fa9218b0de221510d)
- [fix broken tests location_attrs](https://github.com/NoSoft-SA/nspack/pull/360/commits/4c2566ac59b819053ad471df91b8ab79572a11e8)

How to test this code via the UI:

add staging location to a load, from loads grid see new load staging item to view pallets locations
